utils/chain.ts

```diff
@@ -38,3 +38,25 @@ export const getAvailablePrefix = (prefix: string): string => {
     ? prefix
     : ''
 }
+
+export enum Network {
+  KUSAMA = 'kusama',
+  BASILISK = 'basilisk',
+  BASILISK_TESTNET = 'basilisk-testnet',
+  STATEMINE = 'statemine',
+  STATEMINT = 'statemint',
+  POLKADOT = 'polkadot',
+  MOONRIVER = 'moonriver',
+  MOONBEAM = 'moonbeam',
+}
+
+export const networkToPrefix: Record<Network, Prefix> = {
+  [Network.POLKADOT]: 'dot',
+  [Network.KUSAMA]: 'ksm',
+  [Network.STATEMINT]: 'ahp',
+  [Network.BASILISK]: 'bsx',
+  [Network.BASILISK_TESTNET]: 'snek',
+  [Network.STATEMINE]: 'ahk',
+  [Network.MOONRIVER]: 'movr',
+  [Network.MOONBEAM]: 'glmr',
+}
```

Comment:
It may be not necessary to add this stuff. You could check  `chainNames` on `libs/static/src/chains.ts` Could you try to unify them?

Reply:
Removed the extra enum and used a type that was already created inside 
`stores/identity.ts`

<img width="294" alt="CleanShot 2023-08-23 at 11 05 11@2x" src="https://github.com/kodadot/nft-gallery/assets/44554284/05c677af-aa46-456a-ad9f-0b43bb413865">

the map itself it was already present , just moved it to a new file 
<img width="268" alt="CleanShot 2023-08-23 at 11 05 47@2x" src="https://github.com/kodadot/nft-gallery/assets/44554284/46e71485-242d-46cd-9e42-e642dcf30bf9">

let me know if you still want me to change this

Reply:
OK. Let's just remove the enum and modify the map like this

```
export const networkToPrefix: Record<ChianType, Prefix> = {
```